commands/Tools/contact.js

```diff
@@ -0,0 +1,71 @@
+exports.init = async (client) => {
+  const schema = client.settings.guilds.schema;
+  if (!schema.bugChannelId) {
+    await client.settings.guilds.add("bugChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.ideaChannelId) {
+    await client.settings.guilds.add("ideaChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.messageChannelId) {
+    await client.settings.guilds.add("titleURL", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.reportChannelId) {
+    await client.settings.guilds.add("reportChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+};
+
+exports.run = async (client, msg, [type, ...content]) => {
+  // Declares all necessary variables to use for the command.
+  const { bugChannelId, ideaChannelId, messageChannelId, reportChannelId } = msg.guild.settings;
```

Comment:
I could not find another method to doing so. I thought you guys did not like adding variables on the commands itself. In my bot, these variables are in the config.json file so my bot can use them easily but if there is a better way I would be happy to change them and recommit.